### ISSUE TITLE
Roles and Permissions - correct?

### ISSUE THREAD

**Issue by icgaff** [MicrosoftDocs/entra-docs#206]
I have a user with User Administrator role who doesn't have access to group expiration settings in the tenant. Is this article wrong? Is it just Global Administrator, for example that can set expiration?

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: ee217532-6b64-61f7-d965-bc95d9198cdb
* Version Independent ID: f6bab9a9-1b8b-01db-6e42-045eb04efe9e
* Content: [Set expiration for Microsoft 365 groups - Microsoft Entra ID](https://learn.microsoft.com/en-us/entra/identity/users/groups-lifecycle)
* Content Source: [docs/identity/users/groups-lifecycle.md](https://github.com/MicrosoftDocs/entra-docs/blob/main/docs/identity/users/groups-lifecycle.md)
* Service: **active-directory**
* Sub-service: **enterprise-users**
* GitHub Login: @barclayn
* Microsoft Alias: **barclayn**

**Comment by SaibabaBalapur-MSFT** [comment]
@icgaff 
Thanks for your feedback! We will investigate and update as appropriate.

**Comment by ManoharLakkoju-MSFT** [comment]
Hi @icgaff 
According to Microsoft documentation, to configure the expiration policy for Microsoft 365 groups, you need to have one of the following roles:

- Global Administrator
- Group Administrator
- User Administrator

If a user with the User Administrator role does not have access to the group expiration settings in the tenant, it could be due to a permission issue. You can check if the user has the necessary permissions to configure the expiration policy by following these steps:

1. Sign in to the Azure portal with a Global Administrator account.
2. Go to Azure Active Directory > Users > User settings.
3. Under the "Administrative roles" section, click "Add assignments".
4. Select the User Administrator role and add the user to the role.
5. Click "Save" to save the changes.

After adding the user to the User Administrator role, they should have access to the group expiration settings in the tenant. If the issue persists, you can check if there are any custom role assignments or policies that are restricting the user's access to the group expiration settings.

**Comment by icgaff** [comment]
Thanks. I've tested with a new user account, assigned the user
administrator role and the user administrator has access via the GUI for a
short period of time before getting an error accessing the blade. Tested
this with two tenants. Access is no problem through Powershell with User
Administrator however so we'll access the settings that way. I'm sure in
time someone at MS with the ability to fix it notices.

Chris.
